BlockServer/test_modules/test_runcontrol_manager.py

```diff
@@ -79,7 +73,7 @@ def _create_initial_runcontrol_manager(self):
         rcash = Mock()
         rcm = RunControlManager(prefix, "", "",
                                 ioc_control, ch,
-                                MockBlockServer(), self.cs, dummy_sleep_func, run_control_auto_save_helper=rcash)
+                                MockBlockServer(), self.cs, run_control_auto_save_helper=rcash)
```

Comment:
I would rather hand this in than patch because patching requires sleep to be public and if it isn't public then you shouldn't really be touching it in a test. However, I do not have that strong a feeling about this.

Reply:
As discussed, it's more pythonic to use patching rather than passing dummy classes.